contracts/kleros/KlerosGovernor.sol

```diff
@@ -104,16 +104,16 @@ contract KlerosGovernor is Arbitrable {
      *  @param _description The string in CSV format that contains labels of list's transactions.
      *  Note that the submitter may give bad descriptions of correct actions, but this is to be seen as UI enhancement, not a critical feature and that would play against him in case of dispute.
      */
-    event ListSubmitted(uint indexed _listID, address indexed _submitter, uint _session, string _description);
+    event ListSubmitted(uint indexed _listID, address indexed _submitter, uint indexed _session, string _description);
 
     /** @dev Constructor.
-     *  @param _arbitrator The arbitrator of the contract. It should support appealPeriod.
+     *  @param _arbitrator The arbitrator of the contract. It must support appealPeriod.
```

Comment:
We should be clearer on this. Either it is required or not.

Reply:
These words are interchangeable to me but "must" would be more gramatically correct, yeah